content/pages/resources.md

```diff
@@ -0,0 +1,29 @@
+Title: Resources
+Category: pages
+Summary: Feminism101 and Other Resources
+
+# Feminism101 #
+
+[Geek Feminism 101](http://http://geekfeminism.wikia.com/wiki/Feminism_101)
```

Comment:
I'd like to include Julie Pagano's article here if possible! http://juliepagano.com/blog/2013/11/02/101-off-limits/